vehicle/accel_brake_map_calibrator/include/accel_brake_map_calibrator/accel_brake_map_calibrator_node.hpp

```diff
@@ -56,8 +57,8 @@
 namespace accel_brake_map_calibrator
 {
 
-using autoware_vehicle_msgs::msg::SteeringReport;
-using autoware_vehicle_msgs::msg::VelocityReport;
+using autoware_auto_vehicle_msgs::msg::SteeringReport;
+using autoware_auto_vehicle_msgs::msg::VelocityReport;
```

Comment:
We will not use autoware_auto_msgs in the future, so these changes are unnecessary.